### DISCUSSION TITLE
Increase verbosity / Show compilation flags

### DISCUSSION THREAD

**Discussion by skoudoro** [mesonbuild/meson-python#505] [General]
Hi meson-python Team. 

Thank you for this package. is it possible to increase verbosity?

When I do `pip install -vvv .` I would like to see compilation flags like (`clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/include -fPIC -O2 -isystem /opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/include -I/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/include/python3.9 -c test.c -o test.o -msse2 -mfpmath=sse` )

but currently, it is hidden.

is there a way to do that? if not, should I create an issue ?

Thank you for the feedback!

**Comment by rgommers** [comment]
Yes, there is. Passing `-v` to `pip` will cause it to show the meson build output, but that doesn't show compilation flags by default. So in adding you need to use the `-v`/`--verbose` option to `meson compile`. So this will work:
```
# replace -C by --config-settings= for pip versions older than 23.1
pip install . -v -Ccompile-args=-v
```

**Reply by skoudoro** [reply]
Great ! thank you for the feedback!
